iib/web/models.py

```diff
@@ -930,8 +955,14 @@ def from_json(cls, kwargs, batch=None):
             request_kwargs[key] = [
                 Image.get_or_create(pull_specification=item) for item in request_kwargs.get(key, [])
             ]
-
+        build_tags = request_kwargs.pop('build_tags', [])
         request = cls(**request_kwargs)
+        db.session.add(request)
+        db.session.flush()
+
+        for bt in build_tags:
```

Comment:
You have a function to do this. `add_build_tag`

Reply:
yes, that's why I'm using it. Or I don't get your point

Reply:
Sorry I think I misunderstood something here.